lib/Request.js

```diff
@@ -11,7 +11,7 @@ class Request extends CallableInstance {
     super('send')
 
     this._remit = remit
-    this._emitter = new EventEmitter()
+    this._remit._emitter = this._remit._emitter || new EventEmitter()
```

Comment:
We'll still need `this._emitter` as we still want events only relevant to this particular request instance.